### DISCUSSION TITLE
If VSCode autoclose brackets copilot can't understand it

### DISCUSSION THREAD

**Discussion by csiszaralex** [community/community#18969] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/38698577/174580316-1217b26b-1932-4b61-abca-7b0ef4e5f25f.png)

**Comment by charleprr** [comment, marked as answer]
This looks good to me.

Copilot is suggesting what the content of the line should be, even if it means keeping a bracket open.
It will close the first bracket on one of the lines below, in a later suggestion.
